Source/Microsoft.Gateway/Microsoft.InnerEye.Gateway.Models/ProcessorSettings.cs

```diff
@@ -31,6 +45,13 @@ public class ProcessorSettings : IEquatable<ProcessorSettings>
         /// </summary>
         public Uri InferenceUri { get; }
 
+        /// <summary>
+        /// Get the license key based on the environment variable.
+        /// </summary>
+        [JsonIgnore]
+        public string LicenseKey =>
+            Environment.GetEnvironmentVariable(LicenseKeyEnvVar, EnvironmentVariableTarget.Machine) ?? string.Empty;
```

Comment:
should this throw if the env var does not exist?

Reply:
I don't think so, it's needed for display in the installer and used to create the client. I think this should be a dumb lookup. There is a warning printed to the log before the inference client is constructed.